connectors/filtering/basic_rule.py

```diff
@@ -147,6 +147,50 @@ def from_string(cls, string):
                 )
 
 
+class BasicRuleValidationResult:
+    def __init__(self, rule_id, is_valid, validation_message):
+        self.rule_id = rule_id
+        self.is_valid = is_valid
+        self.validation_message = validation_message
+
+    @classmethod
+    def valid_result(cls, rule_id):
+        return BasicRuleValidationResult(
+            rule_id=rule_id, is_valid=True, validation_message="Valid rule"
+        )
+
+
+class BasicRuleValidator:
```

Comment:
No need to review (temporary copy from https://github.com/elastic/connectors-python/pull/268)